src/SFA.DAS.EmployerIncentives.Web/Views/Payments/_PaymentStatus.cshtml

```diff
@@ -7,32 +7,39 @@
 
 @if (Model != null)
 {
-    @if (!Model.LearnerMatchFound && !Model.PausePayments)
+    @if (Model.PaymentIsStopped)
     {
-        <p class="govuk-body govuk-!-margin-bottom-0">Training provider needs to confirm they have started their apprenticeship</p>
+        <p class="govuk-body govuk-!-margin-bottom-0" data-linktype='view-agreement-stopped'>Apprenticeship paused or stopped</p>
```

Comment:
The data-linktype attribute is generally used with anchor tags, maybe something like data-paragraphtype would be more appropriate here?

Reply:
Good point.  Updated